### PR TITLE
Update send-email-as-distribution-list.md

### PR DESCRIPTION
**Pull request by msbemba**
Added instructions for outlook client as well.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4907

File: microsoft-365/admin/manage/send-email-as-distribution-list.md [hunk 1]
```diff
@@ -44,3 +46,13 @@ Before you perform these steps, make sure you've been added to a Microsoft 365 d
 5. Right-click on the From address - such as `Ina@weewalter.me` - and choose **Remove**.<br/> ![Remove the FROM alias](../../media/9b8d8e8f-dc46-499c-89bd-0a480603bf1f.png)
   
 6. Then type the distribution list address such as support@contoso.com, and send the message. The next time you reply from the distribution list, its address will appear as an option in the **From** list.<br/>![Alias of the shared mailbox appears](../../media/f7632a9a-9cab-446c-9e37-23ef50c5b975.png)
+
+## Outlook
+
+1. Open Outlook desktop client.
+
+2. Compose a New Email. Click the **From** field and select **Other email address**. If you do not see the From field, navigate to Options and select From in the section Show fields.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/send-email-as-distribution-list.md]:
```suggestion
2. Compose a New Email. Click the **From** field and select **Other email address**. If you do not see the From field, navigate to **Options** and select **From** in the Show fields section.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**